docs/user/security/audit-logging.asciidoc

```diff
@@ -3,34 +3,169 @@
+Use the {kib} audit logs in conjunction with {ref}/enable-audit-logging.html[{es} audit logging] to get a
+holistic view of all security related events. {kib} defers to the {es} security
+model for authentication, data index authorization, and features that are driven
+by cluster-wide privileges. For more information on enabling audit logging in
+{es}, refer to {ref}/auditing.html[Auditing security events].
 
 [IMPORTANT]
 ============================================================================
-Audit logs are **disabled** by default. To enable this functionality, you
-must set `xpack.security.audit.enabled` to `true` in `kibana.yml`.
+Audit logs are **disabled** by default. To enable this functionality, you must
+set `xpack.security.audit.enabled` to `true` in `kibana.yml`.
 ============================================================================
 
-Audit logging uses the standard {kib} logging output, which can be configured
-in the `kibana.yml` and is discussed in <<settings>>.
+The current version of the audit logger uses the standard {kib} logging output,
+which can be configured in `kibana.yml`. For more information, refer to <<settings>>.
+The audit logger uses a separate logger and can be configured using
+the options in <<audit-logging-settings>>. 
 
 ==== Audit event types
 
-When you are auditing security events, each request can generate 
-multiple audit events. The following is a list of the events that can be generated:
+When you are auditing security events, each request can generate multiple audit
+events. The following is a list of the events that can be generated:
 
 |======
 | `saved_objects_authorization_success`    | Logged when a user is authorized to access a saved
                                              objects when using a role with <<kibana-privileges>>
 | `saved_objects_authorization_failure`    | Logged when a user isn't authorized to access a saved
                                              objects when using a role with <<kibana-privileges>>
 |======
+
+==== ECS audit events
+
+beta[]
+
+[IMPORTANT]
+============================================================================
+The following events are only logged if the audit logger is enabled.
+For information on how to configure `xpack.security.audit.appender`, refer to
+<<audit-logging-settings>>.
+============================================================================
+
+Refer to the table of events that can be logged for auditing purposes. 
+
+Each event is broken down into `category`, `type`, `action` and `outcome` fields
+to make it easy to filter, query and aggregate the resulting logs. 
+
+[NOTE]
+============================================================================
+To ensure that a record of every operation is persisted even in case of an
+unexpected error, asynchronous write operations are logged immediately after all
+authorization checks have passed, but before the response is received.
```

Comment:
I stuck for a few seconds before realized **response from Elasticsearch** is meant. Is it worth clarifying?

Reply:
Yes, agree this makes it clearer